src/Data/OsgNode.cpp

```diff
@@ -425,10 +425,12 @@ osg::ref_ptr<osg::Node> Data::OsgNode::createNodeSphere( const float& scaling, o
 
 	osg::ref_ptr<osg::Geode> geode = new osg::Geode;
 	geode->addDrawable( nodeSphere );
+    geode->setNodeMask(0x2);
 
 	osg::ref_ptr<osg::AutoTransform> at = new osg::AutoTransform();
 	at->setAutoRotateMode( osg::AutoTransform::ROTATE_TO_SCREEN );
 	at->addChild( geode );
+    //at->setNodeMask(0x2);
```

Comment:
treba alebo netreba ???
ak netreba zmazat,
ak mozno bude treba kvoli inym castiam kodu, tak to treba okomentovat